sklearn/manifold/mds.py

```diff
@@ -129,18 +131,18 @@ def _smacof_single(similarities, metric=True, n_components=2, init=None,
     return X, stress, it + 1
 
 
-def smacof(similarities, metric=True, n_components=2, init=None, n_init=8,
+def smacof(dissimilarities, metric=True, n_components=2, init=None, n_init=8,
```

Comment:
On the con side, the new prototype is kind of confusing if you use tab completion... I am not sure how we should handle this case.